components/expenses/PayExpenseBtn.js

```diff
@@ -127,18 +127,24 @@ class PayExpenseBtn extends React.Component {
             }
           `}
         </style>
-        <SmallButton className="pay" onClick={this.onClick} disabled={this.props.disabled || disabled} title={title}>
+        <StyledButton
+          className="pay"
+          buttonStyle="success"
+          onClick={this.onClick}
+          disabled={this.props.disabled || disabled}
+          title={title}
+        >
           {selectedPayoutMethod === 'other' && (
-            <FormattedMessage id="expense.pay.manual.btn" defaultMessage="record as paid" />
+            <FormattedMessage id="expense.pay.manual.btn" defaultMessage="Record As Paid" />
           )}
           {selectedPayoutMethod !== 'other' && (
             <FormattedMessage
               id="expense.pay.btn"
-              defaultMessage="pay with {paymentMethod}"
+              defaultMessage="Pay With {paymentMethod}"
```

Comment:
My example in https://github.com/opencollective/opencollective/issues/2570 (`Mark as paid`) was using sentence case, sorry for confusing it by saying title case

```suggestion
              defaultMessage="Pay with {paymentMethod}"
```